src/main/java/core/Main.java

```diff
@@ -14,28 +14,30 @@
 import java.time.LocalDateTime;
 
 public class Main {
-    private static final Injector injector
+    private static final Injector INJECTOR
```

Comment:
although this is static and final - this is not a magic number or string, or something like that
let's change the name back to `lowercase`

Reply:
Done